lldb/source/Commands/CommandObjectDWIMPrint.cpp

```diff
@@ -182,11 +184,13 @@ void CommandObjectDWIMPrint::DoExecute(StringRef command,
                                      expr);
       }
 
-      bool failed = errorToBool(dump_val_object(*valobj_sp));
-      if (!failed)
+      Error err = dump_val_object(*valobj_sp);
+      if (!err)
         return;
 
       // Dump failed, continue on to expression evaluation.
+      LLDB_LOG_ERROR(GetLog(LLDBLog::Expressions), std::move(err),
```

Comment:
I'm confused about the order of the arguments to this macro. The error comes first, then the format string, then the other parameter?